### PR TITLE
Feat(ERC20 transaction fees)

### PR DESCRIPTION
**Pull request by schnogz**
## Description
- Adds ability to fetch and display ERC20 Tx fees from new /v2 ETH tx endpoint.  
- PAX transaction list will now fetch and display tx fee in terms of ETH when the sent transaction is expanded.

## Change Type
- Feature

## Testing Steps
Detail the steps required for the reviewer(s) to verify and test these changes.

## Code Checklist
- [x] Code compiles successfully (verified via `yarn start`)
- [x] `README.md` and other documentation is updated as needed

### CODE REVIEW

**Comment by plondon** [comment]
Whats wrong with all the locales?

**Comment by schnogz** [comment]
> Whats wrong with all the locales?

What do you mean?

**Comment by schnogz** [comment]
oh i see what you mean. im not exactly sure why/how those changed.  locales still work though

File: packages/blockchain-wallet-v4-frontend/src/components/TransactionListItem/index.js [hunk 1]
```diff
@@ -11,7 +11,12 @@ class ListItemContainer extends React.PureComponent {
   state = { isToggled: false }
 
   handleToggle = () => {
+    const { coin, ethTxActions, erc20List, transaction } = this.props
+
     this.setState({ isToggled: !this.state.isToggled })
+    if (this.state.isToggled && includes(coin, erc20List)) {
+      ethTxActions.fetchTransaction(transaction.hash)
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/components/TransactionListItem/index.js]: https://gyazo.com/debb2711b27a95e968aa6c574e4a7f3f

Toggling transaction on PAX throws this error

**Pull request merged**